[IMP] Purchase: update reporting (#10400)
odoo/documentation: content/applications/inventory_and_mrp/purchase/advanced/analyze.rst

   decision has had on the amount of money they have spent on Bike Friends products.

   David starts by opening Odoo and navigating to :menuselection:`Purchase --> Reporting -->
   Purchase`. Then, he selects the :icon:`oi-view-pivot` :guilabel:`(pivot table)` option at the top
   of the screen.

   In the search bar, he types `Bike Friends`, and clicks :guilabel:`Enter`, so the report only
   shows data for purchases from Bike Friends.

   Then, David opens the search bar drop-down menu. In the :guilabel:`Confirmation Date` field, he
   leaves the :guilabel:`June` and :guilabel:`2024` filters enabled. He also selects
   :guilabel:`Confirmation Date: Previous Period` in the :guilabel:`Comparison` section.

   Next, David clicks on the :guilabel:`Measures` :icon:`fa-caret-down` drop-down menu. He leaves
   the :guilabel:`Total` and :guilabel:`Untaxed Total` datasets enabled, and disables the
   :guilabel:`Order` and :guilabel:`Count` datasets.

   Finally, David clicks the :icon:`fa-minus-square-o` :guilabel:`Total` button above the rows on
   the pivot table, and selects the :guilabel:`Product` option.

>    With all of these options configured, the Purchase report shows a pivot table comparing purchase

Reviewer: ```suggestion
   With all of these options configured, the :guilabel:`Purchase Analysis` report shows a pivot table comparing purchase
```

   data for the current month, June, with the previous month, May.

   The pivot table is broken down into two main columns: one for the untaxed total spent, and one
   for the taxed total spent. These columns are further broken down into three smaller columns: the
   amount spent in May, the amount spent in June, and the variation between the two months,
   represented as a percentage.

>    On the left side of the pivot table, one row is shown for each product purchased from Bike
>    Friends during June. Using this report, David is able to see that Bike Haus has spent much less
>    money overall on products purchased from Bike Friends, compared to the previous month.

Reviewer: ```suggestion
   On the left side of the pivot table, one row is shown for each product purchased from *Bike
   Friends* during June. Using this report, David is able to see that *Bike Haus* has spent much less
   money on products purchased from *Bike Friends*, compared to the previous month.
```

> =================
> Purchase Analysis
> =================

Reviewer 2: Please either add "report" or use full sentence case in the H1 :)
```suggestion
========================
Purchase Analysis report
========================
```

```suggestion
=================
Purchase analysis
=================
```

- :guilabel:`Days to Receive`: The number of days it takes to receive the products in a |PO|.
- :guilabel:`Gross Weight`: The total weight of purchased products.
- :guilabel:`Qty Billed`: The quantity of a product (or products) for which the vendor has already
  been billed.
- :guilabel:`Qty Ordered`: The quantity of a product (or products) ordered.
- :guilabel:`Qty Received`: The quantity of an ordered product (or products) received.
- :guilabel:`Qty to be Billed`: The quantity of an ordered product (or products) for which the
  vendor has yet to be billed.
- :guilabel:`Total`: The total amount spent, including tax.
- :guilabel:`Untaxed Total`: The total amount spent, excluding tax. This measure is selected by
  default.
- :guilabel:`Volume`: The total volume of ordered products, for products which are measured by
  volume.
- :guilabel:`Count`: The total count of |POs|.

.. tip::
   Only one measure can be selected at a time when one of the :icon:`fa-area-chart`
   :guilabel:`(graph view)` options is enabled. However, multiple measures, and varying group-by
   criteria (on the x and y axes), can be selected when using the :icon:`oi-view-pivot`
   :guilabel:`(pivot table)`.

> .. _purchase/advanced/analyze/example:

Reviewer 2: In custom anchors, only the parent folder is optional to include, and with it being named "advanced", I think we can omit it, and just use a little more context :)
```suggestion
.. _purchase/purchase-analysis-example:
```

   selects the :guilabel:`Days to Receive` option.

   With all of these options enabled, the :guilabel:`Purchase Analysis` report shows a bar chart,
   with one bar for each vendor, representing the average number of days it takes to receive
   products purchased from the vendor.

   Using this data, David can see that it takes Bike Friends over 4.5 days, on average, to deliver
   purchased products. This is more than four times the amount of time it takes any other vendor.

   Based on these findings, David makes the decision to reduce the quantity of products purchased
   from Bike Friends.

   .. image:: analyze/dtr.png
      :align: center
      :alt: The Purchase report, showing the average days to receive products from vendors.

Use case: compare vendor POs for two time periods
=================================================

Another use for the :guilabel:`Purchase Analysis` report is to compare key statistics about |POs|
for two different time periods, for a specific vendor. By doing so, it is easy to understand how
purchases from the vendor have increased or decreased.

.. example::

>    Following the :ref:`previous example <purchase/advanced/analyze/example>`, it has been one month

Reviewer 2: ```suggestion
   Following the :ref:`previous example <purchase/purchase-analysis-example>`, it has been one month
```

   the :guilabel:`Total` and :guilabel:`Untaxed Total` datasets enabled, and disables the
   :guilabel:`Order` and :guilabel:`Count` datasets.

   Finally, David clicks the :icon:`fa-minus-square-o` :guilabel:`Total` button above the rows on
   the pivot table, and selects the :guilabel:`Product` option.

   With all of these options configured, the :guilabel:`Purchase Analysis` report shows a pivot
   table comparing purchase data for the current month, June, with the previous month, May.

   The pivot table is broken down into two main columns: one for the untaxed total spent, and one
   for the taxed total spent. These columns are further broken down into three smaller columns: the
   amount spent in May, the amount spent in June, and the variation between the two months,
   represented as a percentage.

   On the left side of the pivot table, one row is shown for each product purchased from Bike
   Friends during June. Using this report, David is able to see that Bike Haus has spent much less
   money on products purchased from Bike Friends, compared to the previous month.

   .. image:: analyze/comparison.png
      :align: center

>       :alt: The Purchase report, comparing the amount spent at a vendor,

Reviewer 2: Update to a period :)
```suggestion
      :alt: The Purchase report, comparing the amount spent at a vendor.
```
